### ISSUE TITLE
TF2.2.0 model.train&evaluate may have a display bug 

### ISSUE THREAD

**Issue by kurikinton105** [tensorflow/tensorflow#40575]
Please make sure that this may be a bug. 
**System information**

- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): 
No. This is the code from (https://www.tensorflow.org/beta/tutorials/quickstart/beginner)

- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Google Colab and mac OS Mojave  10.14.4

- TensorFlow version (use command below): pip install tensorflow==2.2.0

- Python version: 3.6.9 

**Describe the current behavior**
There is a difference of result between TF2.1.0 and TF2.20 version in model.train  & model.evaluate sections.  
In TF2.1.0, result of model.train show 60000/60000.it is normal.
But in TF2.2.0, this show 1875/1875  . Despite the same shape of train data.

In case of evaluate is same.
In TF2.1.0, result of model.evaluate show 10000/10000.
But in TF2.2.0, this show 313/313.

This is may be bug.But it's also may normal.
So please check the problem.

**Source code / logs**

```
#install Tensorflow in Colab
!pip install tensorflow==2.2.0
import tensorflow as tf
print(tf.__version__)

#RUN the code 
from __future__ import absolute_import, division, print_function, unicode_literals
import tensorflow as tf

mnist = tf.keras.datasets.mnist

(x_train, y_train), (x_test, y_test) = mnist.load_data()
x_train, x_test = x_train / 255.0, x_test / 255.0
model = tf.keras.models.Sequential([
  tf.keras.layers.Flatten(input_shape=(28, 28)),
  tf.keras.layers.Dense(128, activation='relu'),
  tf.keras.layers.Dropout(0.2),
  tf.keras.layers.Dense(10, activation='softmax')
])

model.compile(optimizer='adam',
              loss='sparse_categorical_crossentropy',
              metrics=['accuracy'])
print("fit") 
model.fit(x_train, y_train, epochs=1)
print("evaluate")
model.evaluate(x_test,  y_test)
```

[out]
Downloading data from https://storage.googleapis.com/tensorflow/tf-keras-datasets/mnist.npz
11493376/11490434 [==============================] - 0s 0us/step
fit
1875/1875 [==============================] - 3s 2ms/step - loss: 0.2957 - accuracy: 0.9136
evaluate
313/313 [==============================] - 0s 2ms/step - loss: 0.1342 - accuracy: 0.9594
[0.13421817123889923, 0.9593999981880188]

I uploaded to [Colab](https://colab.research.google.com/drive/1MxqnCskd43a9-cr3Gjv6Oaby04OzyDcc?usp=sharing )
This ipynb file uses the TF2.1.0 and TF2.2.0. You can see the bug on TF 2.2.0.

**Comment by ravikyram** [comment]
I have tried in colab with TF version 2.2, nightly versions and was able to reproduce the issue.Please, find the gist [here](https://colab.research.google.com/gist/ravikyram/2fc7cca38390186678550f8036c27a2a/untitled45.ipynb).Thanks!

**Comment by ymodak** [comment]
@kurikinton105  This is by design. The `batch_size` argument in `model.fit` if unspecified will default to 32. You can validate this by simply looking at `number of samples (60000)/1875 = batch_size (32)` 
See https://www.tensorflow.org/api_docs/python/tf/keras/Model#fit arguments documentation.
Thanks!

**Comment by kurikinton105** [comment]
Thank you for your answering!!!!!
I checked the document and I understand this design changes.

This changes may be sudden change.
So users may be wondering may be wondering if they can learn from the data properly because there is no display.
Thanks!!!

**Comment by ymodak** [comment]
Glad to help. Closing now. Thanks for confirming.
